src/components/Menu.jsx

```diff
@@ -1,11 +1,29 @@
-import React from 'react'
+import React, { useState, useEffect } from "react";
+import { Segment, List } from "semantic-ui-react";
+import MenuItem from "./MenuItem";
+import axios from "axios";
 
 const Menu = () => {
+  const [menuItems, setIdentification] = useState([]);
```

Comment:
shouldn't `setIdentification` be named the more conventional `setMenuItems`?

Reply:
Correct changed